impl_core/src/main/java/io/opencensus/implcore/stats/MeasureMapImpl.java

```diff
@@ -72,6 +78,20 @@ public void record() {
 
   @Override
   public void record(TagContext tags) {
+    recordInternal(RecordUtils.getTagMap(tags));
+  }
+
+  @Override
+  public void recordWithTags(List<Tag> tags) {
```

Comment:
There should be a version that takes an explicit TagContext as well that this one should be written in terms of.

Reply:
Make sense, added.